Workbooks/Resource Insights/AzureDataExplorerCluster/ResourceInsights.workbook

```diff
@@ -201,40 +207,369 @@
+                    "formatter": 4,
+                    "formatOptions": {
+                      "min": 0,
+                      "palette": "red"
+                    },
+                    "numberFormat": {
+                      "unit": 17,
+                      "options": {
+                        "style": "decimal",
+                        "useGrouping": false,
+                        "maximumFractionDigits": 2
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.cache/redis--cachemisses Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.cache/redis--errors",
+                    "formatter": 4,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 17,
+                      "options": {
+                        "style": "decimal",
+                        "maximumFractionDigits": 1
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.cache/redis--errors Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-TotalNumberOfExtents Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-TotalNumberOfExtents",
+                    "formatter": 1,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 0,
+                      "options": null
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-TotalNumberOfThrottledCommands Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-TotalNumberOfThrottledCommands",
+                    "formatter": 1,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 0,
+                      "options": null
+                    }
+                  }
+                ],
+                "rowLimit": 10000,
+                "hierarchySettings": {
+                  "treeType": 1,
+                  "groupBy": [
+                    "Subscription"
+                  ],
+                  "expandTopLevel": true,
+                  "finalBy": "Name"
+                },
+                "labelSettings": [
+                  {
+                    "columnId": "Subscription"
```

Comment:
need labels here for loc, and generally anywhere that has these label settings that we want translated should have a 'label' field

Reply:
These weren't being used. I removed them.